cli/start.go

```diff
@@ -127,12 +137,16 @@ func runStart(cmd *cobra.Command, args []string) {
 		return
 	}
 
-	err = s.Start(false)
-	if err != nil {
+	if err := s.Start(false); err != nil {
 		log.Errorf("cockroach server exited with error: %s", err)
 		return
 	}
 
+	client := makeDBClient()
+	if err := testutils.SetDefaultRangeReplicaNum(client, 1); err != nil {
```

Comment:
Doesn't this need to be behind `if Context.TransientSingleNode`? 

We shouldn't include `testutils` in the production binary; if we need this it should go in a different package (maybe just `client`?)